src/components/chats/chat/ChatMessages.vue

```diff
@@ -211,36 +412,50 @@ export default {
       this.currentMedia = this.medias.find((el) => el.url === url);
       this.isFullscreen = true;
     },
-
     nextMedia() {
       const imageIndex = this.medias.findIndex((el) => el.url === this.currentMedia.url);
       if (imageIndex + 1 < this.medias.length) {
         this.currentMedia = this.medias[imageIndex + 1];
       }
     },
-
     previousMedia() {
       const imageIndex = this.medias.findIndex((el) => el.url === this.currentMedia.url);
       if (imageIndex - 1 >= 0) {
         this.currentMedia = this.medias[imageIndex - 1];
       }
     },
 
+    handleScroll() {
+      const { chatMessages } = this.$refs;
+      if (!chatMessages) return;
+
+      // if (this.isEdgeRoomMessage(this.roomMessagesSorted[0].minutes)) {
```

Comment:
Dead code?

Reply:
No, this code is commented to handle future updates with the backend